### ISSUE TITLE
PC reboots during `okteto down`

### ISSUE THREAD

**Issue by ro0NL** [okteto/okteto#2696]
**Describe the bug**
My PC reboots during `okteto -n .. -c .. down`,

The  project ran successfully in the foreground using `okteto -n .. -c .. up`

**Desktop (please complete the following information):**

- OS: Ubunutu
- Version 20.04

**Additional context**

I've updated to okteto 2.3, but experienced the same issue on 2.2

**Comment by ro0NL** [comment]
this seems to be caused by a `php-fpm` processing in the foreground during entrypoint

fixed it with `php-fpm -D`

**Comment by ro0NL** [comment]
i figured the container needs to keep running for the filesync to work.

even a simple

```
command: |
    while true; do sleep 1; done
```

makes it crash

**Comment by ro0NL** [comment]
here some relevant debug logs

```
time="2022-05-20T19:58:28+02:00" level=info msg="executing command over ssh: 'sh -c 'while true; do sleep 1; done\n''" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0

time="2022-05-20T19:59:11+02:00" level=debug msg="Received deployment  <<<BLOB>>>

time="2022-05-20T19:59:11+02:00" level=info msg="exiting by applyToAppsChan: application has been modified" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="starting shutdown sequence" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="sent cancellation signal" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="stopping syncthing" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="ssh forward 0.0.0.0:38361->0.0.0.0:8384 -> done" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="terminating syncthing 1138318 without wait" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="terminated syncthing 1138318 without wait" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="stopping forwarders" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="ssh client for exec closed" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="command failed: wait: remote command exited without exit status or exit signal" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="terminal restored" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="ssh forward 0.0.0.0:36863->0.0.0.0:22000 -> done" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="stopped k8s forwarder" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="stopped SSH forward manager" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="completed shutdown sequence" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="activate failed with: synchronization service is disconnected" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="waiting for shutdown sequence to finish" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="activating development container retry=true" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="starting shutdown sequence" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="sent cancellation signal" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="stopping forwarders" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="completed shutdown sequence" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
time="2022-05-20T19:59:11+02:00" level=info msg="finished okteto -n xxx up -l debug" action=ee9ac5cd-d8ad-4df4-a85b-e3690dff375f version=2.3.0
```

```

**Comment by pchico83** [comment]
@ro0NL is the container crashing? or the PC? if it's the dev container, could you update the title of the issue?

**Comment by pchico83** [comment]
which `command` do you have in the okteto manifest?

**Comment by ro0NL** [comment]
@pchico83 i was able to reproduce based on the [`hello-world`](https://kubernetes.io/docs/tutorials/hello-minikube/#create-a-deployment) kubernetes pod, with the following minimal manifest

```yaml
dev:
    hello-node:
        command: |
            while true; do echo 'OK'; sleep 1; done
```

then `okteto up`, all good:

![image](https://user-images.githubusercontent.com/1047696/169827723-b977f3e2-9f48-4ad8-9a9b-b5443ec54474.png)

![image](https://user-images.githubusercontent.com/1047696/169827814-78a1084c-dc64-45d1-ab1a-2bb8df99791e.png)


while running, in  a new shell `okteto down` and poof, my PC reboots

**Comment by ro0NL** [comment]
things work as expected if i manually interrupt the entrypoint command

```
OK
OK
^Csignal: interrupt
roland@roland:~/test$ okteto down
 i  Using default @ ... as context
 ✓  Development container deactivated
```

**Comment by ro0NL** [comment]
then it's looks either machine specifc, or cluster-setup specific. Neither i can share easily :sweat_smile: 

i will try some other machines soonish.

for what it's worth we're running minikube v1.25.2 and k8 v1.20.7

**Comment by dkrystki** [comment]
Not sure if it's relevant to this but for me okteto was causing reboots after calling `okteto down` right after quitting okteto shell (in my utililty script).
I fixed it by adding 2s sleep before calling `okteto down`

**Comment by jLopezbarb** [comment]
@dkrystki Could you post your reproduction steps/OS Information? 
We have been trying to reproduce the issue for a while with no luck so far.

**Comment by dkrystki** [comment]
uname output: Linux q 5.15.0-39-generic 42-Ubuntu SMP Thu Jun 9 23:42:32 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
Ubuntu 22.04 LTS

sourcing the following file in bash shell reproduces it for me

```
okteto up
okteto down
```

**Comment by ro0NL** [comment]
meanwhile i've also bumped hardware / upgraded to ubuntu 22.04, but the issue remains on latest okteto version.

It got worse in the sense after invoking `down` i just get a black terminal screen, and i need to physically hold the power button to trigger reboot.

The logs are the same:

```
time="2022-07-29T12:08:03+02:00" level=debug msg="Received deployment 'api' event: {MODIFIED
<BLOB>
time="2022-07-29T12:08:03+02:00" level=info msg="exiting by applyToAppsChan: application has been modified" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="starting shutdown sequence" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="sent cancellation signal" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="stopping syncthing" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="ssh forward 0.0.0.0:41493->0.0.0.0:8384 -> done" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="ssh client for exec closed" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="command failed: wait: remote command exited without exit status or exit signal" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="terminal restored" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="ssh forward 0.0.0.0:42721->0.0.0.0:22000 -> done" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="terminating syncthing 22061 without wait" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="terminated syncthing 22061 without wait" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="stopping forwarders" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="stopped k8s forwarder" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="stopped SSH forward manager" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="completed shutdown sequence" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="activate failed with: synchronization service is disconnected" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="waiting for shutdown sequence to finish" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="ssh reverse forward 0.0.0.0:9003<-0.0.0.0:9003 -> failed to close: EOF" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="activating development container retry=true" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="ssh reverse forward 0.0.0.0:9003<-0.0.0.0:9003 -> done" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="starting shutdown sequence" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="sent cancellation signal" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="stopping forwarders" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="completed shutdown sequence" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
time="2022-07-29T12:08:03+02:00" level=info msg="finished okteto -n  -c etrias-dev -l debug up" action=28903073-5499-4c8c-b4d4-88410aeef261 version=2.5.0
```

**Comment by ro0NL** [comment]
removed the stop target from our Makefile :)
